src/main/java/com/linecorp/armeria/internal/logging/DropwizardMetricConsumer.java

```diff
@@ -100,29 +99,19 @@ private static boolean isSuccess(ResponseLog res) {
                res.attr(ResponseLog.RPC_RESPONSE).get().getCause() == null;
     }
 
-    private static String method(RequestLog log) {
-        if (log.hasAttr(RequestLog.RPC_REQUEST)) {
-            return log.attr(RequestLog.RPC_REQUEST).get().method();
-        }
-
-        if (log.hasAttr(RequestLog.HTTP_HEADERS)) {
-            HttpHeaders headers = log.attr(RequestLog.HTTP_HEADERS).get();
-            return headers.path() + '#' + headers.method();
-        }
-
-        return MoreObjects.firstNonNull(log.method(), "__unknown__");
-    }
-
-    private DropwizardRequestMetrics getRequestMetrics(String methodLoggedName) {
+    private DropwizardRequestMetrics getRequestMetrics(RequestContext ctx, RequestLog req) {
         return methodRequestMetrics.computeIfAbsent(
-                methodLoggedName,
-                m -> new DropwizardRequestMetrics(
-                        m,
-                        metricRegistry.timer(MetricRegistry.name(m, "requests")),
-                        metricRegistry.meter(MetricRegistry.name(m, "successes")),
-                        metricRegistry.meter(MetricRegistry.name(m, "failures")),
-                        metricRegistry.counter(MetricRegistry.name(m, "activeRequests")),
-                        metricRegistry.meter(MetricRegistry.name(m, "requestBytes")),
-                        metricRegistry.meter(MetricRegistry.name(m, "responseBytes"))));
+                req,
```

Comment:
sorry just notice after reviewed, first is request log has proper equals/hashcode? and second is if RequestLog has too many pattern, the map will have too many key and redundant creation of DropwizardRequestMetrics? Should we use metricName as key?